corehq/apps/app_manager/models.py

```diff
@@ -5545,6 +5545,8 @@ class LinkedApplication(Application):
     # This is the id of the master application
     master = StringProperty()
 
+    pulled_from_master_version = IntegerProperty()
```

Comment:
nit: how about `pulled_master_version` or just `master_version`?

Reply:
I like `pulled_from_master_version`. It tells you exactly what it is.

Reply:
Keeping the original, explicit name.